pkg/templates/kubechainsaw/template_test.go

```
@@ -0,0 +1,306 @@
+package kubechainsaw
+
+import (
+	"os"
+	"testing"
+
+	kcModels "github.com/ugiordan/kube-chainsaw/pkg/models"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)